resources/js/features/homepage/components/citylist/Charts.tsx

```
@@ -0,0 +1,34 @@
+
+    return (
+        <div className="w-full max-w-9/10 backdrop-blur-md p-4 pt-4 pr-8 rounded-2xl overflow-hidden border border-white/30 bg-black/10 flex flex-col gap-4 shadow-xl shadow-white/10">
+            <h1 className="text-center uppercase text-xl">
+                Daylight in minutes
+            </h1>
+            <SunlightBar data={savedCities} step={step} />
+            <div className="flex justify-end gap-4">
+                <button
+                    onClick={() => setStep(step === 1 ? 7 : 1)}
+                    className="border border-white/40 px-4 py-2 rounded-4xl h-10 cursor-pointer disabled:opacity-50 disabled:cursor-not-allowed text-sm text-center hover:border-white hover:bg-black/50 active:scale-90 transition-all duration-150 ease-in"
+                >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)